### PR TITLE
Don't allow for renaming on homepage for demo orgs

### PR DESCRIPTION
**Pull request by sampoder**
## Summary of the problem

Ruby would freak out and not load the page

## Describe your changes

Does a check to make sure it isn't demo mode.

### CODE REVIEW

File: app/views/hcb_codes/_memo.html.erb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <% form ||= false %>
 <% prepended_to_memo ||= "" %>
 
-<% if organizer_signed_in?(hcb_code.event) %>
+<% if hcb_code.event&.demo_mode? == false && organizer_signed_in?(hcb_code.event)  %>
```
Comment (sampoder) [inline app/views/hcb_codes/_memo.html.erb]: The "&" is needed because our fake hcb codes don't have events it seems

**Review by sarthaktexas — APPROVED** [review]
lgtm thanks sam

**Review by sarthaktexas — APPROVED** [review]
lgtm thanks sam

File: app/views/hcb_codes/_memo.html.erb [hunk 2]
```diff
@@ -3,7 +3,7 @@
 <% form ||= false %>
 <% prepended_to_memo ||= "" %>
 
-<% if organizer_signed_in?(hcb_code.event) %>
+<% if hcb_code.event&.demo_mode? == false && organizer_signed_in?(hcb_code.event) %>
```
Comment (garyhtou) [inline app/views/hcb_codes/_memo.html.erb]:
```suggestion
<% if organizer_signed_in?(hcb_code.event) && !hcb_code.event&.demo_mode? %>
```

**Review by garyhtou — APPROVED** [review]

File: app/views/hcb_codes/_memo.html.erb (hunk 2 above)
Comment (sampoder) [reply app/views/hcb_codes/_memo.html.erb]: I don't think this will work because when event is nil it'll be true

**Pull request merged**